nikola/plugins/command/import_wordpress.py

```diff
@@ -248,12 +251,16 @@ def _find_wordpress_compiler(self):
         """Find WordPress compiler plugin."""
         if self.wordpress_page_compiler is not None:
             return
+
         plugin_info = self.site.plugin_manager.get_plugin_by_name('wordpress', 'PageCompiler')
-        if plugin_info is not None:
-            if not plugin_info.is_activated:
-                self.site.plugin_manager.activatePluginByName(plugin_info.name)
-                plugin_info.plugin_object.set_site(self.site)
-            self.wordpress_page_compiler = plugin_info.plugin_object
+        if plugin_info is None:
+            candidates = self.site.plugin_manager.locate_plugins()
+            wordpress_candidates = [c for c in candidates if c.name == "wordpress" and c.category == "PageCompiler"]
+            if wordpress_candidates:
+                new_plugins = self.site.plugin_manager.load_plugins(wordpress_candidates)
+                for p in new_plugins:
+                    self.site._activate_plugin(p)
+                    self.wordpress_page_compiler = p
```

Comment:
We still need
```suggestion
                    self.wordpress_page_compiler = p
        else:
            self.wordpress_page_compiler = p
```
here.

Reply:
It is, and it should be applied to both values. I committed the fix manually.